pkg/cmd/server/start/master_args.go

```diff
@@ -62,6 +64,8 @@ func BindMasterArgs(args *MasterArgs, flags *pflag.FlagSet, prefix string) {
 
 	flags.Var(&args.NodeList, prefix+"nodes", "The hostnames of each node. This currently must be specified up front. Comma delimited list")
 	flags.Var(&args.CORSAllowedOrigins, prefix+"cors-allowed-origins", "List of allowed origins for CORS, comma separated.  An allowed origin can be a regular expression to support subdomain matching.  CORS is enabled for localhost, 127.0.0.1, and the asset server by default.")
+
+	flags.StringVar(&args.SchedulerConfigFile, prefix+"scheduler-config-file", args.SchedulerConfigFile, "File with scheduler policy configuration")
```

Comment:
Is this something that we expect to be set often outside of production environments?  We're trying to keep the set of flags down the minimum reasonable set for development, demo, and kicking the tires sort of purposes.  All the options that we have to have, but are more advanced are required to be set inside of a config file (we can generate a default one).

@liggitt Do you know enough about this option to provide advice?